torq.q

```diff
@@ -477,7 +477,7 @@ loadf0:{[reload;x]
   if[not[reload]&x in loadedf;.lg.o[`fileload;"already loaded ",x];:()];
   .lg.o[`fileload;"loading ",x];
   // error trapped loading of file
-  @[system;"l ",x;{.lg.e[`fileload;"failed to load ",x," : ",y]}[x]];
+  $[.z.q;@[system;"l ",x;{.lg.e[`fileload;"failed to load ",x," : ",y]}[x]];system"l ",x];
```

Comment:
Okay sounds good. I believe `.proc.params` should contain `debug` in its key when debug is run.  Would a condition as **`debug in key .proc.params** be suitable for this case?  Or am I missing an issue with this condition?

Reply:
yep I think that should work - this function is inside the `.proc` namespace I think, so torq convention would be to just do `` `debug in key params`` rather than the fully qualified name

obviously test and make sure it works as expected (i.e. in non-debug mode, encountering an error in loading a file should log an error & exit; in debug mode it should put you into a debug prompt at the relevant location in the loaded file)